nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -191,17 +191,23 @@ class AbstractRNNTDecoding(ConfidenceMixin):
         blank_id: The id of the RNNT blank token.
     """
 
-    def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
+    def __init__(self, decoding_cfg, decoder, joint, blank_id: int, num_extra_outputs: int = 0):
         super(AbstractRNNTDecoding, self).__init__()
         self.cfg = decoding_cfg
         self.blank_id = blank_id
+        self.num_extra_outputs = num_extra_outputs
```

Comment:
This line becomes self.joint.num_extra_outputs then.

Reply:
Done.